Tensile/KernelWriterAssembly.py

```diff
@@ -7046,10 +7088,18 @@ def localReadDo(self, kernel, bufferIdx, iui, epsi, tP):
         numReadsPerVector = 1 # kernel["MatrixInstN"] * kernel["MatrixInstB"] // globalParameters["WavefrontWidth"]
       numReadsAlongK = int((kernel["MatrixInstK"] * tP["bpe"]) // (blockWidth*self.bpr)) # TODO:
 
-
+    loopIdx = self.unrollIdx
+    loopDim = kernel["ProblemType"]["IndicesSummation"][loopIdx]
+    loopChar = self.indexChars[loopDim]
     # mfma: for AB tile in NT layout
     if kernel["MatrixInstruction"] and numReadsAlongK > 1:
       tmpVgpr = self.vgprPool.checkOut(2)
+      maskVgpr = None
+      isLastProductSgpr = self.getTmpSgpr(2)
+      if kernel["MatrixInstruction"] and \
```

Comment:
There might be a merge conflict here, you should get latest code

Reply:
latest code pushed